torch/csrc/jit/frontend/builtin_functions.cpp

```diff
@@ -49,6 +49,12 @@ def ndim(a : Tensor) -> int:
   return a.dim()
 def T(a : Tensor) -> Tensor:
   return a.numpy_T()
+def H(a : Tensor) -> Tensor:
+  return a.matrix_H()
+def mT(a : Tensor) -> Tensor:
+  return a.mT
+def mH(a : Tensor) -> Tensor:
+  return a.mH
```

Comment:
can we add some tests for these ? When you add properties to tensors it does not manually generate any operators. I think you need to add a `prim::mH` operator to https://github.com/pytorch/pytorch/blob/7fff21afcf2f62d1a44ab8c5c5aebf25174b7d30/torch/csrc/jit/runtime/register_prim_ops.cpp. 


See: https://github.com/pytorch/pytorch/issues/30775 although I don't think that should be closed..

Reply:
Where should these tests go and how should they look like?
Also, I did not see an entry in that table for `T`, how should the entries for these operations look like? I do not know anything about the internal workings of JIT, sorry.